src/Entity/ShipmentServiceOptions.php

```diff
@@ -91,9 +97,9 @@ public function __construct($response = null)
             }
             if (isset($response->DirectDeliveryOnlyIndicator)) {
                 $this->DirectDeliveryOnlyIndicator = $response->DirectDeliveryOnlyIndicator;
```

Comment:
What I'm a bit afraid of, but not 100% sure, is that the tag is just returned in their XML, but without a value. So $DirectDeliveryOnlyIndicator will become '' or something like that, not really TRUE or FALSE. Later in the XML generation it will not be included again. This breaks people's functionality parsing UPS XML responses and using it again for another POST (not sure if people do that, anyhow). So maybe we should set it to TRUE when the indicator is present, if we go for the true/false approach.